example/src/App.tsx

```diff
@@ -123,6 +138,55 @@ class App extends React.Component<{}, State> {
             </OffCanvasMenu>
           </OffCanvas>       
         <div>
+          <h1>This is my Modal Component!!</h1>
+          <Modal
+            close
+            closeOnBackgroud
+            closeOnEsc
+            modalOverflow
+            backdropEnabled
+            header="This is my modal header"
+            footer={<Button>ok</Button>}
+            activator={<Button>Modal</Button>}
+            size="Medium"
+            show={this.state.show}
+            trigger={{
+              body: 'Open',
+              animate: {
+                in: (modal: object, dialog: object) => this.animateIn(modal, dialog),
+                out: (modal: object, dialog: object) => this.animateOut(modal, dialog),
```

Comment:
Binding arrow function in JSX, or the render function is a really bad practice.  https://medium.com/@machnicki/handle-events-in-react-with-arrow-functions-ede88184bbb#278f  Instead, setup the animate object outside of render method.